client/fvRsConsIf_service.go

```diff
@@ -0,0 +1,48 @@
+package client
+
+import (
+	"fmt"
+
+	"github.com/ciscoecosystem/aci-go-client/models"
+)
+
+func (sm *ServiceManager) CreateContractInterface(tnVzCPIfName string, application_epg string, application_profile string, tenant string, description string, nameAlias string, fvRsConsIfAttr models.ContractInterfaceAttributes) (*models.ContractInterface, error) {
```

Comment:
Is it for the ContractInterface or the ContractInterfaceRelationship? I would adapt the name to include Relationship